src/config.h

```diff
@@ -2,7 +2,7 @@
 #define _config_h_
 
 // app parameters
-#define DEBUG 0
+#define DEBUG 1
```

Comment:
What is the point of this debug flag being checked in here? Is this required for your feature?

Reply:
no it was changed to increase commit count